File: rest/connect-apps/instance-get-example-1/instance-get-example-1.5.x.rb
# Get twilio-ruby from twilio.com/docs/ruby/install
require 'twilio-ruby'

# Get your Account SID and Auth Token from twilio.com/console
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'

# Initialize Twilio Client
@client = Twilio::REST::Client.new account_sid, auth_token

# Get an object from its sid. If you do not have a sid,
# check out the list resource examples on this page
@connect_app = @client.account
                      .connect_apps('CNb989fdd207b04d16aee578018ef5fd93').fetch

@connect_app.update(description: 'This is a demo app.')

puts @connect_app.description

File: rest/connect-apps/instance-get-example-1/instance-get-example-1.5.x.py
# Download the Python helper library from twilio.com/docs/python/install
from twilio.rest import TwilioRestClient

# Your Account Sid and Auth Token from twilio.com/user/account
account_sid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
auth_token = "your_auth_token"
client = TwilioRestClient(account_sid, auth_token)

connect_app = client.connect_apps.get("CNb989fdd207b04d16aee578018ef5fd93")
print(connect_app.homepage_url)
